debian/install

```diff
@@ -10,3 +10,5 @@ lxpanel-plugin/*.png usr/share/kano-updater/images/
 
 icon/updater.app usr/share/applications/
 icon/kano-updater.png usr/share/icons/Kano/66x66/apps/
+
+locale/* /usr/share/locale
```

Comment:
Do a find /usr/locale and you will immediately understand the structure of these .mo files.